public/scripts/app.js

```diff
@@ -82,9 +85,16 @@
       .find("form")
       .on("click", (e) => {
         const rating = e.target.value;
-        console.log(rating);
-
-        // We need to do something with the value!!
+        db.query(`SELECT COUNT(*) FROM resource_ratings WHERE resource_id = $1 AND user_id = $2;`, [req.params.id, req.session.userId])
+        .then((data) => {
+          console.log(data.rows[0]);
+          if (data.rows[0] === 0) {
+            db.query(`INSERT INTO resource_ratings (rating, user_id, resource_id)
+                  VALUES()`)
+          } else { 
+            // db.query('UPDATE resource_ratings ON 
+          }
+        });
```

Comment:
Confused about what we are trying to do here. Clicking on the edit form in `/profile` shouldn't update a resource rating. Clicking the stars on a resource should make a post request using ajax to the API.

If you did want to do a post request from the client side script you would have to post to an api endpoint. The database insert would happen in the api controller. Something like:
```js
$.ajax({ url: '/api/resource/update', method: 'POST', data: $('.profile-container form').serialize() })
```